tests/dotnet-sos/Makefile

```diff
@@ -4,7 +4,8 @@ include $(TOP)/defs.mak
 APPBUILDER=$(TOP)/scripts/appbuilder
 HEAP_SIZE="768M"
 
-MYST_LLDB = ./oelldb/oelldb
```

Comment:
The oelldb make target can be removed now.

Reply:
This is still required by `tests-without-docker`

Reply:
I mean the make target, where we are cloning the repo on L25. GH wouldn't allow a comment there, so added here.

Reply:
Get it. Removed.